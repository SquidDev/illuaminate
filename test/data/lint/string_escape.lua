-- config: (lint (only syntax:string-escape))
local _ = {
  "a",
  "a\nb",
  "a\0b",
  "a\x00b",
  "a\(b",
  "\"\'",

  'a',
  'a\nb',
  'a\0b',
  'a\x00b',
  'a\(b',

  '\"\'',

  "\u{1F991}",
}
