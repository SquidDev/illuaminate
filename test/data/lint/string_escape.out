string_escape.lua: Unknown escape character '\('. [syntax:string-escape]
   │
 7 │   "a\(b",
   │     ^^

string_escape.lua: Unknown escape character '\('. [syntax:string-escape]
    │
 14 │   'a\(b',
    │     ^^
2 errors and 0 warnings
